src/oneD/StFlow.cpp

```diff
@@ -147,6 +147,14 @@ void StFlow::setTransport(Transport& trans)
     m_trans = &trans;
     m_do_multicomponent = (m_trans->transportType() == "Multi");
 
+    if (!m_do_multicomponent && m_do_soret) {
+        throw CanteraError("setTransport",
+            "Thermal diffusion (the Soret effect) is "
+            "enabled, and requires using a multicomponent "
+            "transport model."
+        );
```

Comment:
I was assuming this would fit on fewer lines now. I also prefer the close parentheses on the same line, fwiw (I know, nitpicky).

Reply:
OK I'll fix it up, and yes, I do plan to squash the commits before merging 😄